staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/testdata/test-plugin

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+(>&2 echo "$KUBERNETES_EXEC_INFO")
```

Comment:
Why do we need a sub-shell here? And how it differs from `echo >&2 "$KUBERNETES_EXEC_INFO"` that is easier to understand IMHO.

Reply:
No reason. Updated.